src/css/progressbar.css

```diff
@@ -33,6 +33,27 @@
     transition: clip-path 0.5s ease;
 }
 
+.progressbar.dark-blue>:first-child {
+    background: #0054ae;
+}
+
+.progressbar.green>:first-child {
+    background: #36B24D;
+}
+
+.progressbar.dark-green>:first-child {
+    background: #1d8931;
+}
+
+.progressbar.red>:first-child {
+    background: #FA5352;
+}
+
+.progressbar.dark-red>:first-child {
+    background: #C92B2A;
+}
```

Comment:
The colors are fine but they are not from the default palette and stand out.
This and the default blue should be moved from css file into inline css in jsx and use the MantineTheme colors.
If you are not sure how to do that I can move current styles there and you can build on top of it.